library/general/test/yast2/rel_url_test.rb

```diff
@@ -1,4 +1,6 @@
-require_relative "../test_helper"
+# this mocks the Yast::InstURL module
+require_relative "../../../packages/test/test_helper.rb"
```

Comment:
this looks strange. I expect that mock is in top level test_helper, so you do not need to load mock from another sub module